pkg/controller/dataimportcron-controller.go

```diff
@@ -820,10 +853,16 @@ func (r *DataImportCronReconciler) newDataSource(cron *cdiv1.DataImportCron) *cd
 }
 
 // Create DataVolume name based on the DataSource name + prefix of the digest sha256
-func createDvName(prefix, digest string) string {
+func createDvName(prefix, digest string) (string, error) {
```

Comment:
I suppose you might need to use `util.naming.GetResourceName` to make sure the name is always valid and not too long.

Reply:
Right, added.

Reply:
Just make sure you always use the same name, created with the same method.

Reply:
Sure